Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -2506,6 +2506,37 @@ public IEnumerator ManipulateViaAppBarMotionController()
             yield return null;
         }
 
+        /// <summary>
+        /// Test creating an new instance of a scriptable configuration and setting it.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator SetVisualConfiguration()
+        {
+            BoundsControl boundsControl = InstantiateSceneAndDefaultBoundsControl();
+
+            // Create new scriptable
+            BoxDisplayConfiguration boxDisplayConfiguration = ScriptableObject.CreateInstance<BoxDisplayConfiguration>();
+            yield return null;
+
+            // Set the material property of the new scriptable
+            boxDisplayConfiguration.BoxMaterial = testMaterial;
+            yield return null;
+
+            // Set new scriptable
+            boundsControl.BoxDisplayConfig = boxDisplayConfiguration;
+            yield return null;
+
+            GameObject rigRoot = boundsControl.transform.Find("rigRoot").gameObject;
+            Assert.IsNotNull(rigRoot, "rigRoot couldn't be found");
+
+            Transform boxVisual = rigRoot.transform.Find("box display");
+            Assert.IsNotNull(boxVisual, "box visual couldn't be found");
+
+            // Make sure the new scriptable visuals have been applied to the object
+            Assert.AreEqual(boxVisual.GetComponent<Renderer>().material.color, testMaterial.color);
```

Comment:
Is it worth also verifying that these _aren't_ equal before we set the `BoxDisplayConfig`?

Reply:
Yes it is, added new helper method since the rig is re-created after a new scriptable is set.